scenes/game/game.gd

```diff
@@ -39,6 +39,8 @@ const LocationInfoButtonPair = preload("res://scenes/game/location_infobutton_pa
 @onready var emote_dialog : EmoteDialog = $EmoteDialog
 @onready var modal_dialog : ModalDialog = $ModalDialog
 
+@onready var PopoutLock = Mutex.new()
```

Comment:
not a huge fan of throwing locks onto things without really understanding the problem.
What threads do you think this is an issue with?
The UI should all run on the UI thread anyway, so I don't see how this is going to solve anything.
Does _process handling events maybe simultaneously do something while the UI thread is processing actually showing the thing or something?

The card popout is definitely one of the worst/hackiest things i wrote in this project.

I think this lock probably? won't break anything.
Have you tried running the game in browser mode from godot? There's a button up there to launch it that way

Reply:
Based on cpat's videos, I was thinking that the issues come up when the user and the game logic try to open/close a popup at the same time (e.g. clicking the gauge window when it's about to bring it up to ask for a force/gauge cost); my computer's too fast to reproduce the same behavior, but maybe it would be smarter to just not open a new popup if one's already open? Would probably need to combine with that indicator of the zone it wants you to click, though.

I don't seem to be able to run the game in browser mode, it says something about a missing template file?